test/Microsoft.OpenAPI.OData.Reader.Tests/Common/EdmModelHelper.cs

```diff
@@ -358,7 +358,7 @@ public void MultipleInheritanceEdmModelMetadataDocumentTest()
         <Property Name=""Name"" Type=""Edm.String"" Nullable=""false"" />
       </EntityType>
       <EntityType Name=""Horse"" BaseType=""NS.Animal"" OpenType=""true"">
-        <Property Name=""Height"" Type=""Edm.Decimal"" Nullable=""false"" />
+        <Property Name=""Height"" Type=""Edm.Decimal"" Nullable=""false"" Scale=""Variable"" />
```

Comment:
Was this change necessary in this PR?

Reply:
Yes otherwise I'd get failed test results because if differences. I don't think it's related do much to the change of dotnet version more than the update of the reference.